packages/office-ui-fabric-react/src/components/Tooltip/Tooltip.Props.ts

```diff
@@ -40,6 +40,12 @@ export interface ITooltipProps extends React.HTMLAttributes<HTMLDivElement | Too
   delay?: TooltipDelay;
 
   /**
+   * Max width of tooltip
+   * @default 364px
```

Comment:
Was that in CSS before or is this a behavior change? Like if I'm a current tooltip user and I don't set anything for `maxWidth` will I now get a different `maxWidth` than before just by upgrading?

Reply:
Same max width that was specified in css. And will be same 1 class specificity so shouldn't break someone that manually overrode it